File: coffee.js
var coffee = Math.round(Math.random(0, 10) * (10 - 0 + 1)) + 0;

var showSupply = function () {
	print("*Coffee supply at " + String(coffee) + ".*");
};

var goToStarbucks = function () {
	print("*Drives all the to Augusta to get frappe.*");
	print("*Returns from Augusta.*");

	coffee += 100;

	showSupply();
};

var codeOn = function () {
	print("*Coffee supply all good!*");
	print("*I can now finish my Javascript Nodes.*");

	showSupply();
};

if (coffee <= 5) {
	goToStarbucks();
} else {
	codeOn();
}

File: coffee.java
public class coffee {	
	public static int coffee = (int)(Math.random() * 10);
	
	public static void showSupply() {
		System.out.print("*Coffee supply at ");
		System.out.print(coffee); 
		System.out.println(".*");
	}

	public static void goToStarbucks() {
		System.out.println("*Drives all the to Augusta to get frappe.*");
		System.out.println("*Returns from Augusta.*");
	
		coffee += 10;
	
		showSupply();
	}

	public static void codeOn() {
		System.out.println("*Coffee supply all good!*");
		System.out.println("*I can now finish my Petagon database hacker.*");
	
		showSupply();
	}

	public static void main(String[] args) {
		if (coffee <= 5) {
			goToStarbucks();		
		} else {
			codeOn();
		}
	}
}